File: app/src/main/java/wangdaye/com/geometricweather/db/entities/ChineseCityEntity.java
package wangdaye.com.geometricweather.db.entities;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;


/**
 * Chinese city entity matching GreenDAO schema 62.
 */
@Entity(tableName = "CHINESE_CITY_ENTITY")
public class ChineseCityEntity {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "ID")
    public Long id;

    @ColumnInfo(name = "CITY_ID")
    public String cityId;

    @ColumnInfo(name = "PROVINCE")
    public String province;

    @ColumnInfo(name = "CITY")
    public String city;

    @ColumnInfo(name = "DISTRICT")
    public String district;

    @ColumnInfo(name = "LATITUDE")
    public String latitude;

    @ColumnInfo(name = "LONGITUDE")
    public String longitude;

    public ChineseCityEntity() {
    }
}

File: data/src/main/java/wangdaye/com/geometricweather/db/entities/ChineseCityEntity.kt
package wangdaye.com.geometricweather.db.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlin.jvm.JvmField

/**
 * Chinese city entity matching GreenDAO schema 62.
 */
@Entity(tableName = "CHINESE_CITY_ENTITY")
class ChineseCityEntity {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "ID")
    @JvmField
    var id: Long? = null

    @ColumnInfo(name = "CITY_ID")
    @JvmField
    var cityId: String? = null

    @ColumnInfo(name = "PROVINCE")
    @JvmField
    var province: String? = null

    @ColumnInfo(name = "CITY")
    @JvmField
    var city: String? = null

    @ColumnInfo(name = "DISTRICT")
    @JvmField
    var district: String? = null

    @ColumnInfo(name = "LATITUDE")
    @JvmField
    var latitude: String? = null

    @ColumnInfo(name = "LONGITUDE")
    @JvmField
    var longitude: String? = null
}
